src/components/Base.js

```diff
@@ -2,15 +2,15 @@ import Article from "../components/article/Article";
 import { CSSTransition, TransitionGroup } from "react-transition-group";
 import "../assets/css/animation.css";
 
-function Base({ setArticles, articles }) {
-  function calculateSum() {
-    let price = 0;
-    articles.forEach((article) => {
-      price += parseFloat(article.price.replace(",", ".")) * article.amount;
-    });
-    return price.toFixed(2);
-  }
+export const calculateSum = (articles) => {
+  let price = 0;
+  articles.forEach((article) => {
+    price += parseFloat(article.price.replace(",", ".")) * article.amount;
+  });
+  return price.toFixed(2);
+};
```

Comment:
When you have time for a refactor, you can replace the `forEach` method with the `reduce` method. The `reduce` method was "designed" for "add/subtract" things.

Read more about reduce [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce).